source/fundamentals/collations.txt

```diff
@@ -0,0 +1,311 @@
+
+Consider the following ``findOneAndUpdate()`` operation on this collection
+which **does not** specify a collation:
+
+.. literalinclude:: /code-snippets/collation/findOneAndUpdate-collation.js
+   :language: javascript
+   :start-after: start findOneAndUpdate without collation
+   :end-before: end findOneAndUpdate without collation
+   :dedent: 4
+
+Since "Gunter" is the first sorted result when using a binary collation, none
+of the documents come lexically before and match the ``$lt`` comparison
+operator in the query document. As a result, the operation does not update any
+documents.
+
+Consider the same operation with a collation specified with the locale set to
+``de@collation=phonebook``. This locale specifies the ``collation=phonebook``
+option which contains rules for prioritizing proper nouns, identified by
+capitalization of the first letter. The ``de@collation=phonebook`` locale and
+option sorts characters with umlauts before the same characters without
+umlauts.
+
+.. literalinclude:: /code-snippets/collation/findOneAndUpdate-collation.js
+   :language: javascript
+   :start-after: start findOneAndUpdate with collation
+   :end-before: end findOneAndUpdate with collation
+   :dedent: 4
+
+Since "Günter" lexically comes before "Gunter" using the
+``de@collation=phonebook`` collation specified in ``findOneAndUpdate()``,
+the operation returns the following updated document:
+
+::
+
+  { lastErrorObject: { updatedExisting: true, n: 1 },
+    value: { _id: 3, first_name: 'Günter' },
+    ok: 1 }
+
+findOneAndDelete() Example
+``````````````````````````
+
+The following example calls the ``findOneAndDelete()`` operation on a
+collection that uses the default binary collation and contains the following
+documents:
+
+::
+
+  { "_id" : 1, "a" : "16" }
+  { "_id" : 2, "a" : "84" }
+  { "_id" : 3, "a" : "179" }
+
+In this example, we set the ``numericOrdering`` collation parameter to ``true``
+to sort numeric strings based on their numerical order instead of their
+lexical order.
+
+.. literalinclude:: /code-snippets/collation/findOneAndDelete-collation.js
+   :language: javascript
+   :start-after: start findOneAndDelete example with collation
+   :end-before: end findOneAndDelete example with collation
+   :dedent: 4
+
+After you run the operation above, the collection contains the following
+documents:
+
+::
+
+  { "_id" : 1, "a" : "16" }
+  { "_id" : 2, "a" : "84" }
+
+If you perform the same operation without collation on the original
+collection of three documents, the server deletes the first document it
+finds in which the lexical value in the "``a``" field is greater than the
+lexical value
+of "``100``".
+
+.. literalinclude:: /code-snippets/collation/findOneAndDelete-collation.js
+   :language: javascript
+   :start-after: start findOneAndDelete example without collation
+   :end-before: end findOneAndDelete example without collation
+   :dedent: 4
+
+Since lexical value of all the values of ``a`` ("``16``", "``84``", and
```

Comment:
I missed this before -- you should probably add a "the" here:
```suggestion
Since the lexical value of all the values of ``a`` ("``16``", "``84``", and
```
(and then change "are greater" to "is greater" below)

Reply:
I think the wording of these two paragraphs is still awkward. I'm updating it to provide better clarity.